runtime/oti/j9consts.h

```diff
@@ -482,6 +482,7 @@ extern "C" {
 #define J9_GC_POLICY_METRONOME OMR_GC_POLICY_METRONOME
 #define J9_GC_POLICY_NOGC OMR_GC_POLICY_NOGC
 
+// J9_GC_MINIMUM_OBJECT_SIZE does not include dataAddr field
```

Comment:
A C++ comment is not appropriate for a C header file. I'll create a PR to fix this, otherwise I expect we'll see compiles fail.

Reply:
See https://github.com/eclipse-openj9/openj9/pull/17342.

Reply:
Technically this comment itself is not correct. J9_GC_MINIMUM_OBJECT_SIZE specifies minimum possible object size and has nothing to do with object headers details. I am suggesting to remove this comment at all

Reply:
I can elaborate more:
- `J9_GC_MINIMUM_OBJECT_SIZE` just means GC would not expect end of the object (does not matter mixed or indexable) in address lower than `objectBeginningAddress + J9_GC_MINIMUM_OBJECT_SIZE`. There is only purpose for this definition

Reply:
gotcha, creating a PR to fix it

Reply:
please read comments above, PR has a been created already